src/main/java/roomescape/Controller/WebController.java

```diff
@@ -12,6 +12,10 @@ public String Home() {
 
     @GetMapping("/reservation")
     public String Reservation() {
-        return "reservation";
+        return "new-reservation";
     }
+
+    @GetMapping("/time")
+    public String Time() { return "time"; }
```

Comment:
개행 예쁘게 해주세요~
인텔리제이 사용중이라면 `CMD + Opt + L` 로 라인 포맷팅이 가능합니다

```suggestion
    public String Time() { 
      return "time"; 
    }
```

Reply:
헉 천재님 감사합니다!!
2e95798090c9cf7360cd6c840c82831c110a5d37